frontend/src/Editor/AppVersionsManager.jsx

```diff
@@ -124,8 +124,10 @@ export const AppVersionsManager = function AppVersionsManager({
   };
 
   const selectVersion = (version) => {
-    setEditingAppVersion(version);
-    setAppDefinitionFromVersion(version);
+    appVersionService.getOne(appId, version.id).then((data) => {
+      // setEditingAppVersion(data);
```

Comment:
can remove since the component would re-render right?

Reply:
done ✅